### DISCUSSION TITLE
What if we had a 4 directional option?

### DISCUSSION THREAD

**Discussion by AstralButOnGitHub** [ScratchAddons/ScratchAddons#5225] [Ideas]
![image](https://user-images.githubusercontent.com/116039257/196451683-f3e66785-2a37-4318-8a09-8d0cf6f1e95b.png)
All it would is when selected it make the desired sprite only rotate towards 0, 90, -180, -90
-----------
Cheers_ Blaze

**Comment by CST1229** [comment]
Wouldn't work for people not using the addon.

**Reply by CST1229** [reply]
This addon would add a new feature to the Scratch editor that affects the project, so it wouldn't work for people not using Scratch Addons or the addon and could break the project. Therefore, it's probably rejected.

**Reply by Samq64** [reply]
I think he means somthing similar to #3699 which wouldn't cause problems for people who don't have Scratch Addons.

**Reply by Samq64** [reply]
Nvm

**Reply by BroJac5246** [reply]
> I think he means somthing similar to #3699 which wouldn't cause problems for people who don't have Scratch Addons.

That's how I'm interpreting it

**Reply by CST1229** [reply]
> I think he means somthing similar to #3699 which wouldn't cause problems for people who don't have Scratch Addons.

Notice how the mockup has a fourth button next to the other rotation styles, which would imply it adds a new rotation syle.

**Reply by BroJac5246** [reply]
> > I think he means somthing similar to #3699 which wouldn't cause problems for people who don't have Scratch Addons.
> 
> Notice how the mockup has a fourth button next to the other rotation styles, which would imply it adds a new rotation syle.

Oh 🤦

**Reply by AstralButOnGitHub** [reply]
> This addon would add a new feature to the Scratch editor that affects the project, so it wouldn't work for people not using Scratch Addons or the addon and could break the project. Therefore, it's probably rejected.

Oh, right

**Reply by AstralButOnGitHub** [reply]
Oh, i forgot that this isn't sscratch
